content/articles/webhooks.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: Webhooks Documentation
+excerpt: You can find our API documentation at the developer site.
+categories:
+- API
+---
+
+# Webhooks Documentation
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+DNSimple uses Webhooks to send notification of the events that happens in your account via HTTP requests.
+
+For more technical information visit the [Webhook and Events section](https://developer.dnsimple.com/v2/webhooks/events/) in the API documentation.
+
+## Find a Webhook
+
+To check your webhooks, log into your account and navigate to your account page.
```

Comment:
```suggestion
To check your webhook recipients, log into your account and navigate to your account page.
```

Reply:
This makes a lot of sense, I like it.

I also felt the lack of consistency in wording between webhook / recipient / endpoint.

This is much better. Thank you!